Update documentation to incorporate ITB

schivmeister opened the pull request:
Please view the markdown rendered directly at https://github.com/meaningfy-ws/semic-styleguide-rdf-validator/tree/feature/SM15-119/docs

schivmeister committed:
Update documentation to incorporate ITB

csnyulas reviewed (approved):
Looks really nice and comprehensive.

Dragos0000 reviewed (changes requested):
readme for SEMIC Conformance Test Suite for OWL and ShaCL should be different from 
ITB SEMIC SHACL VALIDATOR. the readme that will appear on github should be the ITB SEMIC SHACL VALIDATOR please take inspiration from https://github.com/meaningfy-ws/model2owl-validator. This readme is describing the service that will appear on ITB

schivmeister commented:
> readme for SEMIC Conformance Test Suite for OWL and ShaCL should be different from ITB SEMIC SHACL VALIDATOR. the readme that will appear on github should be the ITB SEMIC SHACL VALIDATOR please take inspiration from https://github.com/meaningfy-ws/model2owl-validator. This readme is describing the service that will appear on ITB

Agreed with Dragos to clearly demarcate the two components with two sections. That is, until Costas has a different opinion.

schivmeister committed:
Update README to clearly demarcate ITB service and unit tests

Make two different sections to cover the two components (Java validator
service configuration and Python unit tests), until it is decided
otherwise.

TODO:
- Replace GSheet link with HTML when available
- Add usage/configuration information when finalized
- Add other links similar to Model2OWL (ReST/SOAP)
